### DISCUSSION TITLE
Octernship Task Submission

### DISCUSSION THREAD

**Discussion by NIKU-SINGH** [community/community#51540] [GitHub Education]
I completed the task and pushed the changes on the Feedback branch of the private repo created. But havent got any confirmation regarding the task.
@ashwinexe can you help me out here?

**Comment by ashwinexe** [comment]
@NIKU-SINGH feedback branch is for the maintainers to provide you with feedback. Please push your changes to `main` via a separate PR. 

If you observe closely, feedback PR points from `main` to `feedback` branch which helps tracking changes on `main` but not ideal for submissions by a student.

**Reply by NIKU-SINGH** [reply]
@ashwinexe thanks for answering. Btw how can i get confirmation whether my task has been selected or not?
